src/puppetlabs/ezbake/core.clj

```diff
@@ -442,6 +442,25 @@ Additional uberjar dependencies:
             (get-timestamp-string))
     lein-version))
 
+(defn generate-package-version-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s"
+            (str/replace lein-version #"-SNAPSHOT|-" { "-SNAPSHOT" "" "-" "."}))
+    lein-version))
+
+(defn generate-package-release-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s.%s"
+            (str/replace lein-version #".*-" "0.1")
```

Comment:
I'd love to piggy back the package name change onto this PR, but if not, that's fine